federated-learning/CLOUD.md

```diff
@@ -0,0 +1,59 @@
+# Federated Learning on Google Cloud Platform
+
+This guide will show you how to launch Google Cloud instances to run the federated learning tutorial in the cloud.
+
+If you want to run this part of the tutorial, you will need a [GCP account](https://cloud.google.com/) and to install [Google SDK](https://cloud.google.com/sdk/install).
+
+If not already done you'll need to clone the git repository `https://github.com/tf-encrypted/tf-encrypted`. These commands will be run from within the repo (`cd tf-encrypted/tools/gcp`).
+
```

Comment:
Hey @justin1121 , to run the commands like `./start $INSTANCE_NAMES` are they going have to checkout the branch `master-tf2`?

https://github.com/tf-encrypted/tf-encrypted/tree/master-tf2/tools/gcp

Reply:
Suppose so, we can just copy the script here to make it simpler though.

Reply:
good idea! in fact it might be good to have a copy the gcp scripts in this folder so people just have to clone `tf-world-tutorial` and not tf-encrypted.

cc: @jvmancuso @mortendahl any objections?

Reply:
@yanndupis no, makes sense

Reply:
done!